cpu/cc26x0/include/cc26x0_rfc.h

```diff
@@ -403,7 +403,20 @@ struct __attribute__ ((aligned(4))) radio_op_command_s {
 
 
 typedef struct {
-    radio_op_command_t op;
+    uint16_t commandNo; /* W */
```

Comment:
using a struct adds a memory overhead: it might be one of the reason why ble_setup doesn't work properly.